src/pages/solve-overview/template.html

```diff
@@ -47,7 +47,7 @@ <h1 class="rsTl-feature-title"><a class="rsTl-feature-titleLink track-cta" :href
               <a class="rsTl-featuredLarge-img track-cta" :href="featured.second.alias" :style="{ 'background-image': 'url(' + featured.second.solve_secondary + ')', 'background-position': 'top right' }"></a>
               <div class="rsTl-featuredLarge-text">
                 <span class="rsTl-featuredLarge-type">
-                  <strong>{{ featured.second.field_content_type }}</strong> ({{ featured.second.field_content_length }} {{ ' minute ' + viewAction[featured.second.field_content_type_key] | translate }})
+                  <strong>{{ featured.second.field_content_type | translate }}</strong> ({{ featured.second.field_content_length }} {{ ' minute ' + viewAction[featured.second.field_content_type_key] | translate }})
```

Comment:
I think we need to make the same change here.  (i.e. `{{ '@text minute ' + viewAction[featured.first.field_content_type_key] | translate({'@text': featured.first.field_content_length}) }}`)